console/docable.ts

```diff
@@ -0,0 +1,91 @@
+const decoder = new TextDecoder();
+const encoder = new TextEncoder();
+
+/**
+ * Parse data member doc blocks and signatures and place them in a minimalistic,
+ * JSON format.
+ */
+export class Docable {
+
+  protected filepaths: string[];
+  protected json_output: {[k: string]: string[]} = {};
+  protected output_filepath: string;
+
+  constructor(filepaths: string[], outputFilepath: string) {
```

Comment:
I guess quite ironic, but do't suppose a docblock here just to clarify the params? lol

i mean i can see the purpose of it but thats after understanding how you use this class in the script

Reply:
ed8c527